src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
```

Comment:
The class 'MyHashMap' should not start with an empty line. Please remove the empty line at the beginning of the class definition.